network/p2p/inspector/validation/control_message_validation_inspector_test.go

```diff
@@ -351,12 +354,66 @@ func TestControlMessageValidationInspector_processInspectRPCReq(t *testing.T) {
 		})
 		require.NoError(t, inspector.processInspectRPCReq(inspectMsgReq))
 	})
+
+	t.Run("inspectRpcPublishMessages should disseminate invalid control message notification when invalid pubsub messages count greater than configured RPCMessageErrorThreshold", func(t *testing.T) {
+		inspector, distributor, _, subscriptions, _, sporkID := inspectorFixture(t)
+		// 5 invalid pubsub messages will force notification dissemination
+		inspector.config.RPCMessageErrorThreshold = 4
+		// create unknown topic
+		unknownTopic := channels.Topic(fmt.Sprintf("%s/%s", unittest.IdentifierFixture(), sporkID)).String()
+		// create malformed topic
+		malformedTopic := channels.Topic("!@#$%^&**((").String()
+		// a topics spork ID is considered invalid if it does not match the current spork ID
+		invalidSporkIDTopic := channels.Topic(fmt.Sprintf("%s/%s", channels.PushBlocks, unittest.IdentifierFixture())).String()
+
+		pubsubMsgs := unittest.GossipSubMessageFixtures(10, fmt.Sprintf("%s/%s", channels.TestNetworkChannel, sporkID))
+		pubsubMsgs = append(pubsubMsgs, []*pubsub_pb.Message{
+			{Topic: &unknownTopic},
+			{Topic: &malformedTopic},
+			{Topic: &malformedTopic},
+			{Topic: &invalidSporkIDTopic},
+			{Topic: &invalidSporkIDTopic},
+		}...)
+		expectedPeerID := peer.ID("peerID987654321")
```

Comment:
```suggestion
		expectedPeerID := p2ptest.PeerIdFixture(t)
```
Please revise the peer id instances to use the `PeerIdFixture`.

Reply:
https://github.com/onflow/flow-go/pull/4720/commits/88d31c5292334cc509804f65b8269fa623d9e94d